test/Microsoft.AspNetCore.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -164,7 +164,7 @@ public void MoveTo_CopiesAllItems_AndClears()
             source.MoveTo(destination);
 
             // Assert
-            Assert.True(source.IsEmpty);
+            Assert.True(source.IsEmptyOrWhiteSpace);
```

Comment:
All of the tests are now looking for something different. To match previous checks, need to get the entire content and `Assert.IsEmpty(contentString)`. We really don't want some random thing sneaking whitespace in.